src/mitol/google_sheets/models.py

```diff
@@ -69,3 +69,14 @@ class FileWatchRenewalAttempt(Model):
     date_attempted = DateTimeField(auto_now_add=True)
     result = models.CharField(max_length=300, null=True, blank=True)
     result_status_code = PositiveSmallIntegerField(null=True, blank=True)
+
+
+class EnrollmentChangeRequestModel(TimestampedModel):
```

Comment:
I think we should generalize this to something like `GoogleSheetsRequestModel` and also rename `EnrollmentChangeRequestHandler` -> `GoogleSheetsChangeRequestHandler`